dapps/tezos-provider/src/utils/samples.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    PartialTezosDelegationOperation,
+    PartialTezosOriginationOperation as PartialTezosOriginationOperationOriginal,
+    PartialTezosTransactionOperation,
+    TezosOperationType }
+    from "@airgap/beacon-types";
+
+import { ScriptedContracts } from "@taquito/rpc";
+
+interface PartialTezosOriginationOperation
+  extends Omit<PartialTezosOriginationOperationOriginal, "script"> {
+  script: ScriptedContracts;
+}
+  
+export enum DEFAULT_TEZOS_METHODS {
+  TEZOS_GET_ACCOUNTS = "tezos_getAccounts",
+  TEZOS_SEND = "tezos_send",
+  TEZOS_SEND_TRANSACTION = "tezos_send:transaction",
+  TEZOS_SEND_ORGINATION = "tezos_send:origination",
+  TEZOS_SEND_CONTRACT_CALL = "tezos_send:contract_call",
+  TEZOS_SEND_DELEGATION = "tezos_send:delegation",
+  TEZOS_SEND_UNDELEGATION = "tezos_send:undelegation",
+  TEZOS_SIGN = "tezos_sign",
+}
+
+const tezosTransactionOperation: PartialTezosTransactionOperation = {
+  kind: TezosOperationType.TRANSACTION,
+  destination: "tz3ZmB8oWUmi8YZXgeRpgAcPnEMD8VgUa4Ve", // Tezos Foundation Ghost Baker
+  amount: "100000"
+};
+
+const tezosOriginationOperation: PartialTezosOriginationOperation = {
+  kind: TezosOperationType.ORIGINATION,
+  balance: '1',
+  script: { // This contract adds the parameter to the storage value
+    code: [
+      { prim: "parameter", args: [{ prim: "int" }] },
+      { prim: "storage", args: [{ prim: "int" }] },
+      { prim: "code",
+        args: [[
+            { prim: "DUP" },                                // Duplicate the parameter (parameter is pushed onto the stack)
+            { prim: "CAR" },                                // Access the parameter from the stack (parameter is on top)
+            { prim: "DIP", args: [[{ prim: "CDR" }]] },     // Access the storage value (storage is on the stack)
+            { prim: "ADD" },                                // Add the parameter to the storage value
+            { prim: "NIL", args: [{ prim: "operation" }] }, // Create an empty list of operations
+            { prim: "PAIR" }                                // Pair the updated storage with the empty list of operations
+        ]]
+      }
+    ],
+    storage: { int: "10" }
+  },
+};
+
+const tezosContractCallOperation: PartialTezosTransactionOperation = {
```

Comment:
Add samples for stake, unstake and finalize unstake. They are important. 
they are transaction operation with destination as your own address and entrypoint "stake", "unstake", "finalize_unstake" respectively with Unit parameter value.

Reply:
Added stake, unstake, finalize:
https://github.com/trilitech/web-examples/pull/18/commits/dcea32c466b0012c687370dedf06f55ca923acc0